Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  HKActivitySummaryExtensions.swift
+//  SwifterSwift
+//
+//  Created by Gilbert Lo on 7/3/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(HealthKit)
+import HealthKit
+
+// MARK: - Properties
+@available(watchOS 2.2, *)
+public extension HKActivitySummary {
+
+    /// SwifterSwift: Check if stand goal is met
+    var isStandGoalMet: Bool {
+        let unit = HKUnit.count()
+        let standHours = appleStandHours.doubleValue(for: unit)
+        let goal = appleStandHoursGoal.doubleValue(for: unit)
+        return standHours >= goal
```

Comment:
I don't know this API ... but there is the possibility of not having a goal set? what happens in this situation in `doubleValue` return?

Reply:
In real life, there should be always a goal when fetching `HKActivitySummary` from HealthKit